guides/DevOps/Jenkins.md

```diff
@@ -0,0 +1,10 @@
+# What is Continuous Integration
+Is a set of pratices to allow a system to be compiled,built and tested, development in independent environment.
+
+# Overview
+Jenkins is a Continuous Integration web application used to test and create artifacts of a software project.
```

Comment:
What is CI? Explain CI in a way that even a newbie can understand it 😄

Reply:
done